routes/web.php

```diff
@@ -17,7 +17,7 @@
 
 Route::get('/', function () {
     return view('welcome');
-});
+})->name('map');
```

Comment:
@Tbaile can you check if this is correct?

Reply:
You can name a route in any way you want, it seems fine since it's used in `resources/views/select.blade.php`